bot/exts/moderation/incidents.py

```diff
@@ -97,9 +97,19 @@ async def make_embed(incident: discord.Message, outcome: Signal, actioned_by: di
         colour = Colours.soft_red
         footer = f"Rejected by {actioned_by}"
 
+    day_timestamp = discord_timestamp(incident.created_at, TimestampFormats.DATE)
+    time_timestamp = discord_timestamp(incident.created_at, TimestampFormats.TIME)
```

Comment:
Is there any reasoning behind using this over the option that shows both?

Reply:
I believe I did it to be able to customise the format (add extra words between date and time) but in hindsight it's rather unnecessary, so have change to use the DATETIME option.